packages/server/lib/socket-base.ts

```diff
@@ -474,6 +476,8 @@ export class SocketBase {
               return this.localBus.emit('cross:origin:release:html')
             case 'cross:origin:finished':
               return this.localBus.emit('cross:origin:finished', args[0])
+            case 'url:changed':
```

Comment:
Is this related to cookies or something that was missed previously?

Reply:
No, this is vestigial from a previous implementation. I just missed removing it. Addressed in [`3c558b5` (#22320)](https://github.com/cypress-io/cypress/pull/22320/commits/3c558b503b4437bbd7e476b9b552c2ad1c10e0c4).